hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -222,6 +226,14 @@ public synchronized void setConf(Configuration conf) {
 
     curFunctionCosts= new Double[costFunctions.size()];
     tempFunctionCosts= new Double[costFunctions.size()];
+
+    boolean isBalancerDecisionEnabled = getConf()
```

Comment:
For this and other variables controlling whether or not we record decisions or events, some variant of "record" could be in the variable name to make this plain. This is "record" as verb, not as noun. Consider isBalancerDecisionRecording or isBalancerDecisionRecordingEnabled.  We don't need to call the elements FooBarRecords, but when deciding to put them in a ring buffer or not we could be said to be "recording" FooBar entries or not. Helps with clarity I think but just mentioned for your consideration.